tests/test_jobs.py

```diff
@@ -22,6 +22,8 @@
 
 import datetime
 import unittest.mock
+import json
+import httpretty
```

Comment:
`httpretty` is a third party library. We usually leave a blank line between standard libraries and third-party ones.